lib/server.js

```diff
@@ -116,6 +116,7 @@ var SolidusServer = function( options ){
   router.engine( DEFAULT_VIEW_EXTENSION, handlebars.engine );
   router.set( 'view engine', DEFAULT_VIEW_EXTENSION );
   router.set( 'views', paths.views );
+  router.set( 'trust proxy', true ); // Use the X-Forwarded-* headers: https://expressjs.com/en/guide/behind-proxies.html
```

Comment:
Is this required by Fastly, or any other CDN?

Reply:
Without this, the HTTPS protocol doesn't seem to be detected by Solidus on Heroku, so HTTPS requests end up in redirect loops.